app/forms/signup_form.rb

```diff
@@ -27,6 +28,7 @@ def attributes= attrs
     attrs.each do |k,v|
       self.send("#{k}=", v)
     end
+    puts radio.name.present?
```

Comment:
Please remove all `puts` before committing.

Reply:
I have checked all of my commits, i am pretty sure there are no more